src/amo/utils/index.js

```diff
@@ -133,15 +133,17 @@ export const stripLangFromAmoUrl = ({
   return urlString;
 };
 
-export function getClientConfig(_config: typeof config) {
+export function getClientConfig(
+  _config: typeof config,
+): { [key: string]: any } {
```

Comment:
`any` :(

Reply:
Switched to `mixed`, because it can be pretty much any type.